draft-ietf-httpbis-digest-headers.md

```diff
@@ -32,25 +32,12 @@ normative:
   RFC5843:
   RFC4648:
   RFC5234:
+  RFC6234:
```

Comment:
You can drop this and use `{{!SHA2=RFC6234}}` in the first use of the reference.  You can use the same afterwards, or even just `{{!RFC6234}}`.

Reply:
Cool, thanks! I'll use that syntax in an editorial review of the whole doc.